docs/contributing/style-guide.md

```diff
@@ -1,7 +1,405 @@
 ---
 id: style-guide
+title: Style Guide
+sidebar_label: Style Guide
+description: The Sauce Labs Documentation Style Guide
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Sauce Labs welcomes your contributions to our documentation!
+This document describes the options available for creating content for the site,
+along with some guidelines and conventions.
+
+## Markdown
+
+This site uses Docusaurus version 2 beta.The 
+[Docusaurus Markdown Features documentation](https://docusaurus.io/docs/markdown-features) 
+is interesting, but is missing descriptions of several key features. Markdown Guide provides a more comprehensive description of available 
+[Basic Syntax](https://www.markdownguide.org/basic-syntax) and 
+[Extended Syntax](https://www.markdownguide.org/extended-syntax/) with some helpful tips for how to use it well.
+Markdown is used for its readability, but for anything where you need more control or more complex styling,
+plain HTML may also be used directly in the file.
+
+## Frontmatter
+
+At the top of each docs page, you need to include these things: 
+
+| Variable | Description |
+| ----------- | ----------- |
+|id|A brief string that uniquely identifies the page within its parent folder. The id and the name of the file should be the same.|
+|title|The main title of the page. This value will automatically be rendered using the H1 style at the top of the page.
+|sidebar_label|This is what will appear in the left hand navigation tree for the page.|
+|description (optional)|This is what appears when the page is referenced in a Google search result.|
+
+### Code Example
```

Comment:
I'm not sure this needs to be a heading? It feels like overkill for the floating TOC. Thoughts, @spider-sauce ?

Reply:
I was trying to break up the H2 sections somehow. Might be a better option to use a horizonal line (`---`)

Reply:
Not sure why you want to "break up the H2 sections?" Do you think it makes the floating tOC hard to read when it's just a list of H2s? Or is it in the main body of the page that you object to it? It might be helpful to look at the formatting of a CLI or the saucectl config pages to see how we handle property definitions that include syntax samples: https://docs.saucelabs.com/testrunner-toolkit/saucectl

Reply:
Oh, yes, this it totally the right way to do it, I'll fix

Reply:
I tried, this and didn't get it to look good, so I went with putting everything into a table. Either way the H3s are removed.